wasm_builder/src/lib.rs

```diff
@@ -0,0 +1,126 @@
+use blake2::{Blake2s, Digest};
```

Comment:
This file looks like a collage of the CLI, why not build the cli for wasm?

Reply:
Ok, I just saw your message above, I thought the idea was to give the CLI to both the prover and the verifiers; but apparently I wasn't part of the latest conversations, so do whatever's right.

Reply:
The users of the library on the web need a wasm build not a CLI. Hence this PR.

Reply:
We have both a library and a CLI. If they want to write a wrapper for the library to meet their needs they can do it themselves. We shouldn't need to be wrapping our library for every single library user. That's just my 2 cents.

Reply:
Good point.

Reply:
At this point we have the CIL library, an FFI wrapper for it in C, and CLIs for the prover and verifier. Every wrapper we add we'll have to maintain. I'm not set one way or another, I just want us to be thoughtful about it. I remember @JesseALancaster once argued that we might not distribute the FFI wrapper because of the same reason, and a CLI binary would be easier for the users to integrate with.